### DISCUSSION TITLE
return necessary attributes of a study and its sereis also the instances in one response.

### DISCUSSION THREAD

**Discussion by luxunxiansheng** [dcm4che/dcm4chee-arc-light#4073] [General]
In some cases, it may be a plus  to  get a  patient-study-series-instance   tree by one  qido request rather than walking  through from patient to instance level by level via several qido requests .   

Thanks a lot.

**Comment by vrindanayak** [comment]
See [Search for Instances](https://petstore.swagger.io/index.html?url=https://dcm4che.github.io/dcm4chee-arc-light/swagger/openapi.json#/QIDO-RS/SearchForInstances) using query filter `includefield=all`

**Reply by luxunxiansheng** [reply]
Sorry but this is not what I request aobut the new feature. I want a tree structure but this query returns a flat list.  For the tree,at study level, only study level arributes  needed . At series level, only series level attributes need. At instance level, only instance level attributes needed.  This tree structure can reduce the size of the return json .  For example, for a study,say, with 5 series and each series with 1000 instances, we don't have to contain 5000 copies of study level attributes.
